pandas/core/series.py

```diff
@@ -1070,7 +1065,7 @@ def _set_with(self, key, value):
 
     def _set_labels(self, key, value):
         key = com.asarray_tuplesafe(key)
-        indexer = self.index.get_indexer(key)
+        indexer: np.ndarray = self.index.get_indexer(key)
```

Comment:
Shouldn't need to annotate like this; if anything should set the return of `get_indexer` to np.ndarray

Reply:
yah, im trying to pin down what gets passed to _set_values below.  The check on L1075 is never True, so id like to see if we can rip it out